common/network-options.coffee

```diff
@@ -2,7 +2,7 @@ exports.network = network =
 	message: 'Network Connection'
 	name: 'network'
 	type: 'list'
-	choices: [ 'ethernet', 'wifi' ]
+	choices: [ 'ethernet', 'wifi + ethernet' ]
```

Comment:
We can't do this without a lot of prep upfront first. Lots of CLI (and I suspect image maker) logic checks these literal strings and will fail to stop configuring wifi in images if this isn't exactly `wifi`. These values aren't (or shouldn't be) really presentation data, they're the raw data used to define how the image should be configured.

This should be done in the UI directly imo, to translate from `wifi` into whatever you want to show on the button. I agree this is a more correct name here, and it'd be nice to fix that at the data level but given that we've got loads of unchangeable published device images in the wild with this data embedded in them, changing the format (and having to support both everywhere forever) is a fairly painful thing to do.

Reply:
*facepalm/headbutt wall* Ok cool I'll close this PR